web/diplomacourses.shtml

```diff
@@ -143,7 +143,7 @@
           "/TRUE_EVANGELISM/TRUE_EVANGELISM.shtml">True Evangelism</a></td>

```

Comment:
So, I'm not super sure what's going on with the tests, but if they are complaining about all caps, I see a lot of paths in there that are all caps, like this one, also, lines 139, 128, 125, 121. I might have missed some.